app/src/modules/updateModal.js

```diff
@@ -31,7 +31,6 @@ export default (electron, releaseNotes, updateCallBack, versions) => {
     win.show();
   }, 1000);
   ipcMain.on('update', () => {
-    win.close();
```

Comment:
I think you need to call
`ipcMain.removeAllListeners('update')` 
before 
`ipcMain.on('update', () => {	`
To prevent `updateCallBack();` being called twice in case I close the update window and open it again from the OS menu

https://electronjs.org/docs/api/ipc-renderer#ipcrendererremovealllistenerschannel